Functions/xASL_bids_BIDSifyASLJSON.m

```diff
@@ -192,23 +203,23 @@
 				warning('Did not succeed in repeating PLDs for each TE for Hadamard sequence import');
 			end
 			% Make sure that number of volumes can be divided by the repeated PLDs
-			jsonOut.PostLabelingDelay = repeatedPLDs;
+			jsonOut.PostLabelingDelay = repeatedPLDs; %do we want to repeat the PLDs??
 		end
 	end
 end
 
 % Check for FME Hadamard sequences
 if isfield(jsonOut,'SeriesDescription')
-	isHadamardFME = ~isempty(regexp(char(jsonOut.SeriesDescription),'(Encoded_Images_Had)\d\d(_)\d\d(_TIs_)\d\d(_TEs)', 'once'));
-	if isHadamardFME
+	bHadamardFME = ~isempty(regexp(char(jsonOut.SeriesDescription),'(Encoded_Images_Had)\d\d(_)\d\d(_TIs_)\d\d(_TEs)', 'once'));
+	if bHadamardFME
 		startDetails = regexp(char(jsonOut.SeriesDescription),'\d\d(_)\d\d(_TIs_)\d\d(_TEs)', 'once');
```

Comment:
Same here and throughout, we don't need the `char`

Reply:
Typecasting can make the code more stable though.

Reply:
Done